master/getting-started/openstack/installation/redhat.md

```diff
@@ -137,6 +126,7 @@ through the process.
             ETCD_INITIAL_ADVERTISE_PEER_URLS="http://<public_ip>:2380"
             ETCD_INITIAL_CLUSTER="<hostname>=http://<public_ip>:2380"
             ETCD_INITIAL_CLUSTER_STATE=new
+            ETCD_INITIAL_CLUSTER_TOKEN=`uuidgen`
```

Comment:
I don't think you can run a script in a systemd conf file.  Are you sure this works?